apps/docs/.eslintrc.js

```diff
@@ -13,6 +13,7 @@ module.exports = {
     // The Eslint rule still needs work, but you can
     // enable it to test things out.
     '@stylexjs/valid-styles': 'error',
+    // '@stylexjs/no-unused': 'warn',
```

Comment:
do we want to use `warn` or `error` here?

Reply:
If we follow the behavior in internal codebase, then this should be error

Reply:
I will enable this after this PR lands. There are some issues with dependencies where this doesn't always work consistently.